tests/database.test.js

```diff
@@ -0,0 +1,124 @@
+const app = require('../server/app.js');
+const supertest = require("supertest");
+const http = require('http');
+const mongoose = require('mongoose');
+const User = require('../server/dbSchemas/attendanceSchema.js').User;
+const Course = require('../server/dbSchemas/attendanceSchema.js').Course;
+const SeatingLayout = require('../server/dbSchemas/attendanceSchema.js').SeatingLayout;
```

Comment:
Here you can import all three at once using object deconstruction: (just to simplify the code)
```suggestion
const { User, Course, SeatingLayout } = require('../server/dbSchemas/attendanceSchema.js');
```